source/beamline/AMPseudoMotorControl.cpp

```diff
@@ -136,24 +142,27 @@ QString AMPseudoMotorControl::toString() const
 
 	// Note the connected state of each of the child controls.
 
-	controlConnected += "\n";
-
 	int childCount = childControlCount();
 
-	for (int childIndex = 0; childIndex < childCount; childIndex++) {
-		AMControl *child = childControlAt(childIndex);
+	if (childCount > 0) {
 
-		if (child) {
-			controlConnected += "\t" + child->objectName() + " connected: ";
+		controlConnected += "\n";
```

Comment:
The creation of this string could probably be done in a single QString call using args.